modules/appeals_api/spec/services/appeals_api/pdf_construction/generator_spec.rb

```diff
@@ -170,6 +170,22 @@
           File.delete(generated_pdf) if File.exist?(generated_pdf)
         end
       end
+
+      context 'pdf max length content verification' do
+        include SchemaHelpers
+        let(:max_lengths) { find_max_lengths '../../schemas/v2/200995.json', 'maxLength' }
```

Comment:
Can you use `read_schema(filename, schema_version = 'v1')` here?

Reply:
i do use it inside of the 'find_max_lengths' method

Reply:
My thought was that you could define the schema and pass it to find max lengths like:
```
let(:schema) { read_schema '200995.json', 'v2'}
let(:max_lengths) { find_max_lengths schema, 'maxLength' }
```
...and then you already have your schema passed to find_max_lengths. Just thinking it would be nice to remove the long path shenanigans, since that's what the helper method is for :)